src/common/reconciler_test.go

```diff
@@ -403,3 +403,61 @@ func Test_Reconciler_ServiceNeedsUpdate(t *testing.T) {
 		autopilot.Equals(t, test.result, reconciler.ServiceNeedsUpdate(test.input, &test.service))
 	})
 }
+
+func Test_Reconciler_HandleTools(t *testing.T) {
+	// Arrange
+	var (
```

Comment:
Can we please pull most of this out of `var ()`?

Reply:
@davidbloss will do